File: HHTXT/crash.vb
Public Class crash

    Private Sub Timer1_Tick(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Timer1.Tick
        Me.BringToFront()
    End Sub

    Private Sub crash_KeyUp(ByVal sender As Object, ByVal e As System.Windows.Forms.KeyEventArgs) Handles Me.KeyUp
        If e.KeyCode = Keys.C Then
            Application.Exit()
        End If
    End Sub

    Private Sub crash_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load

    End Sub
End Class

File: HHTXT/crash.cs
using System;
using System.Windows.Forms;

namespace WindowsApplication1
{
    public partial class crash
    {
        public crash()
        {
            InitializeComponent();
        }

        private void Timer1_Tick(object sender, EventArgs e)
        {
            BringToFront();
        }

        private void crash_KeyUp(object sender, KeyEventArgs e)
        {
            if (e.KeyCode == Keys.C)
            {
                Application.Exit();
            }
        }

        private void crash_Load(object sender, EventArgs e)
        {

        }
    }
}